docs/non_containerized_NSFS.md

```diff
@@ -0,0 +1,62 @@
+# Non Containerized NSFS
+
+Running nsfs non containerized is useful for deploying in linux without depending on kubernetes, but requires some steps which are described next.
+
+
+## Build
+
+### Build options - 
+1. noobaa-core S3 Select is enabled. This requires boost shared objects to be present on the machine where noobaa is supposed to be installed. This feature can be disabled if not required (due to additional dependency) by running make rpm BUILD_S3SELECT=0.
+2. In order to build RPM packages for other architectures simply run make rpm CONTAINER_PLATFORM=linux/amd64 or make rpm CONTAINER_PLATFORM=linux/ppc64le
+
+Running the following command will result with an RPM file -
+```sh
+make rpm
+```
+
+
+## Install
+
+### Pre-requisites (S3 select enabled) - 
+1. wget - 
+
+```sh
+yum install wget
+```
+2. boost -
+```sh
+wget https://rpmfind.net/linux/centos/8-stream/AppStream/x86_64/os/Packages/boost-system-1.66.0-13.el8.x86_64.rpm
+wget https://rpmfind.net/linux/centos/8-stream/AppStream/x86_64/os/Packages/boost-thread-1.66.0-13.el8.x86_64.rpm
+rpm -i boost-thread-1.66.0-13.el8.x86_64.rpm
```

Comment:
I think the order here needs to be reversed? First system and then thread or else I am getting
```
[root@a82fc3b93bd8 /]# rpm -i boost-thread-1.66.0-13.el8.x86_64.rpm
warning: boost-thread-1.66.0-13.el8.x86_64.rpm: Header V3 RSA/SHA256 Signature, key ID 8483c65d: NOKEY
error: Failed dependencies:
        boost-system(x86-64) = 1.66.0-13.el8 is needed by boost-thread-1.66.0-13.el8.x86_64
        libboost_system.so.1.66.0()(64bit) is needed by boost-thread-1.66.0-13.el8.x86_64
```

Reply:
hmm that's weird, installing them both like that - 
`rpm -i boost-thread-1.66.0-13.el8.x86_64.rpm boost-system-1.66.0-13.el8.x86_64.rpm` won't cry on it
i'll change it thanks